packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -335,9 +333,14 @@ export async function copyParameterJson(
   );
   const targetParameterFilePath = path.join(parameterFolderPath, targetParameterFileName);
   const sourceParameterFilePath = path.join(parameterFolderPath, sourceParameterFileName);
+  const targetParameterContent = await fs.readJson(sourceParameterFilePath);
+  const appName = ctx.projectSettings!.appName;
+  targetParameterContent["parameters"][resourceBaseName] = {
```

Comment:
let's do nothing if resourceBaseName property does not exist (which means user manually deletes it)

Reply:
ok, but we can also add the resource base name on new JSON file.